mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-publish.adoc

```diff
@@ -0,0 +1,61 @@
+= Publish Messages to an MQTT Broker with MQTT Connector
+
+In the following example, you configure the Anypoint Connector for MQTT (MQTT Connector) *Publish* operation that enables you to publish a message to the desired topic and with a specific quality of service (QoS).
+
+. In the *Mule Palette* view of Studio, select *HTTP > Listener*.
+. Drag *Listener* to the Studio canvas.
+. Set *Path* to `/mqtt3/publish`.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the source in the app.
+. Configure the HTTP Listener global element and click *OK*.
+. In the *Mule Palette* view of Studio, select *MQTT3 > On New Message*.
+. Drag *On New Message* to the Studio canvas.
+. In the *On New Message* configuration screen, optionally change the value of the *Display Name* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the source in the app.
+. In the *MQTT3 Config* window, for *Connection*, select *MQTT3 URL Connection*.
+. For *Client id generator*, select *Client id random suffix generator*.
+. Set the following fields:
++
+* *Client ID*: `smart-bentley-123`
+* *Username*: `usertest`
+* *Password*: `passtest`
+* *URL*: `tcp://127.0.0.1:1883`
+[start=13]
+. Click *OK*.
+. In the *Publish* configuration screen, set the following fields:
++
+* *Topic*: `quotes/terryPratchett`
+* *Message* `#["Stories of imagination tend to upset those without one."]`
```

Comment:
lol

Reply:
OK, I would be remiss if I did not caution you to carefully consider your examples for global impact (examples that you know people around the world will likely understand, rather than regional or cultural examples).
However, Terry Pratchett....